tsdb/db.go

```diff
@@ -360,7 +360,8 @@ func (db *DBReadOnly) Querier(mint, maxt int64) (Querier, error) {
 		blocks[i] = b
 	}
 
-	head, err := NewHead(nil, db.logger, nil, 1, DefaultStripeSize)
+	w := wal.NewNoopWAL(db.dir)
```

Comment:
Not a fan of this.. I think we can make it proper inside it.

Reply:
Head does not take any explicit directory currently, so taking care of it in Head would mean passing an additional directory along with WAL. But thinking about it again, at this particular place, we dont need a WAL/mmapped chunks at all, so I will remove it.